src/AzSK.ADO/Framework/Core/SVT/SVTControlAttestation.ps1

```diff
@@ -28,9 +29,12 @@ class SVTControlAttestation
 		$this.ControlSettings=$ControlSettingsJson = [ConfigurationManager]::LoadServerConfigFile("ControlSettings.json");
 		$this.repoProject.projectsWithRepo = @();
 		$this.repoProject.projectsWithoutRepo = @();
-		if (!$this.AzSKSettings) 
-		{	
-			$this.AzSKSettings = [ConfigurationManager]::GetAzSKSettings();				
+		if (!$this.AzSKSettings)
+		{
+			$this.AzSKSettings = [ConfigurationManager]::GetAzSKSettings();
+		}
+		if ([Helpers]::CheckMember($this.ControlSettings, "EnforceApprovedException") -and ($this.ControlSettings.EnforceApprovedException -eq $true)) {
```

Comment:
also check whether control list is not empty

Reply:
changed as per comment